### PR TITLE
Create a new branch, added  font's folder, improved layout "What do w…

### PR DESCRIPTION
**Pull request by M-fkstv**
…e offer" block according design, made layout "footer" block according design.

Homework done.

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -1,40 +1,88 @@
...
+    margin-right: 285px;
+    margin-bottom: 64px;
+}
 
- }
-.stroke{
-   display: block;
-    margin: auto auto 80px;
+.about {
+    height: 211px;
```
Comment (almuly) [inline styles/styles.css]: It's much better to create one class for footer columns and add width:33,3%. Then you can align it using flex properties

File: styles/styles.css [hunk 2]
```diff
@@ -1,40 +1,88 @@
...
+    font-size: 24px;
+    font-weight: 400;
+    margin-top: 0;
 }
 
-.icons{
-    height: 180px;
-    width:  300px;
-    margin: 0 4px 0 4px;
+.footer {
```
Comment (almuly) [inline styles/styles.css:50]: You need to cover all space with blue background. without gaps

**Review by almuly — CHANGES_REQUESTED** [review]

File: styles/styles.css (hunk 2 above)
Comment (M-fkstv) [reply styles/styles.css:50]: done

File: styles/styles.css (hunk 1 above)
Comment (M-fkstv) [reply styles/styles.css]: done

**Pull request merged**